integration-test/src/main/java/uk/gov/gchq/gaffer/integration/impl/ImportExportIT.java

```diff
@@ -96,23 +95,22 @@ public void shouldImportFromFileThenExportBackOutToDifferentFile() throws Operat
                         .build())
                 .build();
 
+        // When
         final Iterable<? extends String> exportedData = graph.execute(importExportOpChain, getUser());
-
-        //   When
-
         Iterable<String> incomingFile = readFile(INCOMING_FILE_PATH);
         Iterable<String> outgoingFile = readFile(path.toFile().getAbsolutePath());
 
 
-        //  Then
+        // Then
+        assertThat((Iterable<String>) exportedData).containsExactlyInAnyOrderElementsOf(expectedData);
```

Comment:
It might be cleaner to put the cast with the `graph.execute` and not therefore not using the `?` at all.